src/Compute/Compute/Common/ComputeClientInstancViewMethod.cs

```diff
@@ -29,6 +29,14 @@ public static class InstanceViewMethod
             return result;
         }
 
+        public static VirtualMachineExtensionsListResult GetWithInstanceView(this IVirtualMachineExtensionsOperations iVmExtensionOperations,
```

Comment:
Could you name it ListWithInstanceView?

Reply:
@maddieclayton , it should be fixed here: https://github.com/lwajswaj/azure-powershell/commit/1fc53716994dc3f5ff922ddb5eb749670b5004ed